src/components/imageContainer/Index.jsx

```diff
@@ -14,7 +14,7 @@ const ImageContainer = (props) => {
     <Container className="image-container">
       <Row>
         <Col style={{ textAlign: 'center' }}>
-          <Image fluid="true" src={src} style={{ width: '80%', height: '100%' }} />
+          <Image fluid="true" src={src} style={{ width: '80%', height: '80%' }} />
```

Comment:
Could you move the style to a css class and call it here instead of styling inline?